dev/ts/main.ts

```diff
@@ -29,12 +30,12 @@ import type {
   ZoomType,
 } from '../../src/bpmn-visualization';
 import { FlowKind, ShapeBpmnElementKind } from '../../src/bpmn-visualization';
-import { fetchBpmnContent, logDownload, logError, logErrorAndOpenAlert, logStartup } from './utils/internal-helpers';
-import { log } from './utils/shared-helpers';
+import { downloadAsPng, downloadAsSvg } from './component/download';
 import { DropFileUserInterface } from './component/DropFileUserInterface';
 import { SvgExporter } from './component/SvgExporter';
-import { downloadAsPng, downloadAsSvg } from './component/download';
 import { ThemedBpmnVisualization } from './component/ThemedBpmnVisualization';
+import { fetchBpmnContent, logDownload, logError, logErrorAndOpenAlert, logStartup } from './utils/internal-helpers';
```

Comment:
**nitpick:** I don't understand why the import are reorder here. I already mention it in other PR and we agreed to handle it with #2742

Reply:
I don't know why the order was change. Maybe my IDE did that when I added a new import.